.github/workflows/ci.yml

```diff
@@ -664,7 +664,18 @@ jobs:
         env:
           GITHUB_WORKSPACE: ${{ github.workspace }}
           GITHUB_REPOSITORY: ${{ github.repository }}
+          NEEDS_FULL_SETUP: ${{ matrix.needs_full_setup }}
         run: |
+          # Artifacts directory must be created after composer install as it would remove the artifacts directory
+          # This seems a bit strange, if you need to add an artifact at an earlier stage then revalidate that
+          # composer install does actually remove the artifact directory
+          mkdir artifacts
+
+          if [[ NEEDS_FULL_SETUP == "false" ]]; then
+            echo "skipping .env, database, and other full site setup steps"
+            exit 0
```

Comment:
Instead of existing which may mess up the rest of the job, just move the block of code inside the conditional

Reply:
Done